app/javascript/components/applications/application_form/PaperApplicationForm.js

```diff
@@ -59,9 +59,22 @@ const buildPrefValidations = (prefs) => {
   return prefValidations
 }
 
+const validateAnnualIncome = (value) => {
+  if (isNil(value)) {
+    return null
+  } else {
+    if (/[^0-9$,.]/.test(value)) {
+      return 'only accepts numbers, $, commas, and periods.'
```

Comment:
this error message wording is provisional. i have a question out to design for input on the wording here

Reply:
resolved with new wording provided by josh